webui/react/src/services/api.ts

```diff
@@ -79,7 +79,7 @@ export const login = generateApi<Credentials, void>(Config.login);
 // use processApiError.
 export function logout(): DetSwagger.V1LogoutResponse {
   const apiName = arguments.callee.name;
-  return sApi.determinedLogout().catch(e => processApiError(apiName, e));
+  return sAuthApi.determinedLogout().catch(e => processApiError(apiName, e));
```

Comment:
discussed with @hamidzr of dropping `determined` prefix if it's a simple swagger config change.

Reply:
let's leave this for another PR.